app/routes/forum/index.tsx

```diff
@@ -0,0 +1,24 @@
+import loadable from '@loadable/component';
+import { Route, Routes } from 'react-router-dom';
+import PageNotFound from '../pageNotFound';
+import ForumDetail from './components/ForumDetail';
+import ForumEditor from './components/ForumEditor';
+import ThreadDetail from './components/ThreadDetail';
+import ThreadEditor from './components/ThreadEditor';
+
+const ForumList = loadable(() => import('./components/ForumList'));
+
+const ForumRoute = () => (
+  <Routes>
+    <Route index element={<ForumList />} />
+    <Route path=":forumId/threads" element={<ForumDetail />} />
```

Comment:
Now I'm knitpicking at a pretty serious level - but this feels more natural idk
```suggestion
    <Route path=":forumId" element={<ForumDetail />} />
```